src/sentry/incidents/subscription_processor.py

```diff
@@ -479,8 +469,13 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
                         # And the trigger is not yet active
                         metrics.incr(
                             "incidents.alert_rules.threshold.alert",
-                            tags=alert_triggered_tags,
+                            tags={"detection_type": self.alert_rule.detection_type},
                         )
+                        if features.has(
+                            "organizations:workflow-engine-metric-alert-dual-processing-logs",
+                            self.subscription.project.organization,
+                        ):
```

Comment:
nit: pull into its own variable so we don't have to check the feature flag twice here